docker/client/entrypoint.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+# Check if the node_modules directory exists
```

Comment:
Could use this file in the dev container as runtime, just need to know that you need to rebuild if any node modules are added.

Reply:
or don't even bother checking whether it exists... always just install from lockfile

Reply:
yeah sure, I removed it